tests/run-make/llvm-outputs/rmake.rs

```diff
@@ -0,0 +1,20 @@
+// test that directories get created when emitting llvm bitcode and IR
+
+use run_make_support::{cwd, run_in_tmpdir, rustc};
+use std::path::PathBuf;
+
+fn main() {
+    let mut path_bc = PathBuf::new();
+    let mut path_ir = PathBuf::new();
```

Comment:
Suggestion: these variables should not be necessary, we can just define them inside `run_in_tmpdir` with `let path_bc = p.join(...)`.

Reply:
That sounds reasonable.